concept-to-clinic/concept-to-clinic
# #205 Export pdf functionality.

Clicking on the **Report and Export** section, you have an **Export** button that downloads the page as a `.pdf` file.

## Description
Create a pdf file of the section of the site in the **Report and Export** section

## Reference to official issue
Addresses issue #205 

## Motivation and Context
This feature allows downloading of the page information in `.pdf` format.

## How Has This Been Tested?
- Load [http://localhost:8080/](http://localhost:8080/)
- Click on the **Report and Export** menu section
- Click on the **Export** button
- It should download the page information in pdf format

## Output samples
[PDF file with eye candy](https://www.scribd.com/document/364610595/Rsna-Standard-Template-14)
[PDF file without eye candy](https://www.scribd.com/document/364610685/Rsna-Standard-Template-12)
## CLA
- [X] I have signed the CLA; if other committers are in the commit history, they have signed the CLA as well
